<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Managemen Buku</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPengarang">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>481</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputJudul">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimpan">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCari">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNama">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lalu Bisma Kurniawan Haris | F1D022055 </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTahun">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHapus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pengarang</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSimpan"/>
    <addaction name="actionExport_CSV"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSimpan_Data"/>
    <addaction name="actionHapus_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSimpan">
   <property name="text">
    <string>Simpan</string>
   </property>
  </action>
  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionSimpan_Data">
   <property name="text">
    <string>Simpan Data</string>
   </property>
  </action>
  <action name="actionHapus_Data">
   <property name="text">
    <string>Hapus Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
